pages/common/pyenv.md

```diff
@@ -0,0 +1,27 @@
+# pyenv  
+
+> Switch between multiple versions of Python easily.
+
+- List all available commands:
+
+`pyenv commands`
+
+- List all Python versions known to pyenv:
```

Comment:
Following-up: does "known to pyenv" mean "globally installed in the current machine", "installed in the ~/.pyenv/versions directory", both, or neither?